doc/version_history.rst

```diff
@@ -2,6 +2,12 @@
 Version History
 ===============
 
+v0.6.3
```

Comment:
I think these are pretty substantial changes.. consider revving up the minor release number, e.g. `v0.7.0`

Reply:
Updated in the commit: [915bd44](https://github.com/lsst-ts/ts_m2com/pull/16/commits/915bd4406424bf702674b475f644a3b449d15726). Thanks!